crates/eframe/src/web/app_runner.rs

```diff
@@ -184,7 +185,6 @@ impl AppRunner {
     ///
     /// The result can be painted later with a call to [`Self::run_and_paint`] or [`Self::paint`].
     pub fn logic(&mut self) {
-        super::resize_canvas_to_screen_size(self.canvas(), self.web_options.max_size_points);
```

Comment:
Is `max_size_points` no longer respected? If so, we should `#[deprecate]` or delete it, and explain what CSS incantation replaces it